tests/integration/tier0/backup-release/test_backup_release.py

```diff
@@ -1,37 +1,193 @@
 import os
-import platform
+import re
+
+from collections import namedtuple
+
+import pytest
 
 from envparse import env
 
 
-system = platform.platform()
+# TODO move to conftest
```

Comment:
Yes please, move this to conftest if possible so it's implemented only at one place so there is no redundant code.

Reply:
I will do so in a separate PR if that's ok.
That and proper pytest refactoring would be a big bite to take, keeping it here IMHO.